src/renderer/src/layouts/WelcomeWithTabs/WelcomeTabs/index.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import { useNavigate } from 'react-router-dom'
+import { Tabs } from '@renderer/components/Tabs'
+
+export enum WelcomeTabItemType {
+  NEON_ACCOUNT = 'neon-account',
+  HARDWARE_WALLET = 'hardware-wallet',
+  ADDRESS_OR_KEY = 'address-or-key',
+}
+
+type TabItem = {
+  text: string
+  url: string
+  type: WelcomeTabItemType
+}
+
+const tabItems: TabItem[] = [
+  {
+    text: 'neonAccount',
+    url: '/neon-account',
+    type: WelcomeTabItemType.NEON_ACCOUNT,
+  },
+  // TODO: remove this menu item comment when it was implemented
+  // {
+  //   text: 'hardwareWallet',
+  //   url: '/hardware-wallet',
+  //   type: WelcomeTabItemType.HARDWARE_WALLET,
+  // },
+  // TODO: remove this menu item comment when it was implemented
+  // {
+  //   text: 'addressOrKey',
+  //   url: '/address-or-key',
+  //   type: WelcomeTabItemType.ADDRESS_OR_KEY,
+  // },
+]
+
+type TProps = {
+  defaultValue: WelcomeTabItemType
+}
+
+export const WelcomeTabs: React.FC<TProps> = ({ defaultValue }) => {
```

Comment:
React.FC typing includes the children prop, which is not true for this component since it does not render it.

A better way to do this is to never use React.FC and always specify children where you would like to receive them.

Example: 

```
type TProps = {
  children: React.ReactNode
}

export const ExampleComponente = (props: TProps) => {}
```